xocto/pact_testing.py

```diff
@@ -9,6 +10,17 @@
 import requests
 
 
+@dataclasses.dataclass(frozen=True)  # Use kw_only=True once we're on Python 3.10.
```

Comment:
according to the docs kw_only is available since python3.10 so it should work here

Reply:
Our docs state we have compatibility with Python>=3.9. My train of thought was that we can only make use of `kw_only` once we no longer need to provide compatibility with Python 3.9. 🤔 

> This library works with Python 3.9 and above. ([source](https://xocto.readthedocs.io/en/latest/index.html))

https://github.com/octoenergy/xocto/blob/b808a5c769b95fd95a43155e44b938c0eb221dea/pyproject.toml#L8

Reply:
oh, sorry, I misunderstood the comment.